### DISCUSSION TITLE
Key pin usage

### DISCUSSION THREAD

**Discussion by Marco1971Repo** [pschatzmann/arduino-audio-driver#3] [Q&A] [answered]
I've a little sketch use the library up and running
I'm trying to understand how to use button (without use of arduino-audio-tools) in my sketch without success...
Could you add an example on how use button in a sketch?

**Comment by pschatzmann** [comment, marked as answer]
The driver just provides an api to query for the pin numbers for the buttons. The logic to read a button you would need to implement yourself. Either by using digitalRead() and some debouncing logic or by using some pin interrupts. 

Use Google: there should be plenty of tutorials.

**Reply by Marco1971Repo** [reply]
I understant. Thank for the replay i know what to do.
